downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -30,6 +30,12 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
 
 ## Backward-incompatible changes
 
+- A bug that resulted in not giving compilation errors for literal values due to invalidly inferred literal value types has been fixed.
+
+    ```ballerina
+    2f|1 num = 2; // Compilation error now.
```

Comment:
@SasinduDilshara @LakshanWeerasinghe, can we add a few more applicable scenarios here.

Reply:
@LakshanWeerasinghe few examples are

```
2f|3d|6 num = 10; // Compilation error now.
3d|6 num = 10; // Compilation error now.
```

Reply:
Updated.